client/src/Components/RatingsAndReviews/ReviewList/AddReviewModal.jsx

```diff
@@ -71,32 +71,38 @@ var AddReviewModal = (props) => {
 
   var sizeOnChange = (event) => {
       var sizeNum = Number(event.target.value);
-      setCharacteristics({...characteristics, '14' : sizeNum});
+      var sizeId = props.metaData.characteristics.Size.id
+      setCharacteristics({...characteristics, sizeId : sizeNum});
```

Comment:
Nice use of destructuring here. I should get more in the habit of doing so like you did here.